demisto_sdk/commands/common/constants.py

```diff
@@ -255,7 +255,7 @@ def found_hidden_param(parameter_name):
 INDICATOR_FIELDS_DIR = 'IndicatorFields'
 LAYOUTS_DIR = 'Layouts'
 CLASSIFIERS_DIR = 'Classifiers'
-MISC_DIR = 'Misc'
+MISC_DIR = 'Packs/Base/Misc'
```

Comment:
@orhovy @bakatzir why was this changed?
it broke the `init` command

Reply:
It was changed because the directory of reputation files was changed to this path. In order the validate will validate reputation files(catch their regex), this was changed.

Reply:
misc dir was re changed in content and is not in base pack anymore: https://github.com/demisto/demisto-sdk/pull/322

Reply:
You are right but this is not relevant to the `init` command.
Even before the revert this is breaking the command

Reply:
I filed an issue https://github.com/demisto/etc/issues/23293